ACMAS/app/acmas_tables.sql

```diff
@@ -0,0 +1,60 @@
+create table "ACMAS_Web_university"
+(
+    id   bigserial
+        primary key,
+    name varchar(50) not null
+);
+
+alter table "ACMAS_Web_university"
+    owner to hello_acmas;
+
+create table "ACMAS_Web_course"
+(
+    id         bigserial
+        primary key,
+    name       text        not null,
+    code       varchar(20) not null,
+    university text        not null,
+    semster    varchar(30) not null,
+    years      text        not null,
+    test_type  varchar(20) not null
+);
+
+alter table "ACMAS_Web_course"
+    owner to hello_acmas;
+
+
+create table "ACMAS_Web_question"
+(
+    id        bigserial
+        primary key,
+    filename  text not null,
+    question  text not null,
+    "Answers" text not null,
+    "Hash"    text not null
```

Comment:
Also, search engine thing--but we won't need "Hash" anymore do to a change in approach. We will use Elasticsearch.

Reply:
Sounds good! You guys should implement this change in your own issue/PR when you're ready.